spec/adapters/alma_adapter_spec.rb

```diff
@@ -140,13 +140,86 @@
     end
   end
 
-  describe ".get_items_for_bib" do
-    it "returns a Hash" do
-      expect(adapter.get_items_for_bib(unsuppressed)).to be_a Hash
+  describe "#get_items_for_bib" do
+    context "when a record has holdings with notes" do
```

Comment:
Does the fixture you used have the temporary locations scenario we looked at? Could you add that to the context here?

Reply:
It does. I'll add it.